modules/manage/partials/monitor-health.adoc

```diff
@@ -89,15 +89,15 @@ While maximizing the rate of messages moving from producers to brokers then to c
 
 The total throughput of a cluster can be measured by the producer and consumer rates across all topics.
 
-To observe the total producer and consumer rates of a cluster, monitor xref:reference:public-metrics-reference.adoc#redpanda_kafka_request_bytes_total[`redpanda_kafka_request_bytes_total`] with the `produce` and `consume` labels, respectively.
+To observe the total producer and consumer rates of a cluster, monitor xref:reference:public-metrics-reference.adoc#redpanda_rpc_sent_bytes[`redpanda_rpc_sent_bytes`] and xref:reference:public-metrics-reference.adoc#redpanda_rpc_received_bytes[`redpanda_rpc_received_bytes`] with the `kafka` `redpanda_server` label.
 
 ==== Producer throughput
 
 For the produce rate, create a query to get the produce rate across all topics:
 
 [,promql]
 ----
-sum(rate(redpanda_kafka_request_bytes_total{redpanda_request="produce"} [5m] )) by (redpanda_request)
+rate(redpanda_rpc_received_bytes{redpanda_server="kafka"}[$__rate_interval])
```

Comment:
Any comment on dropping `sum`? All series will be present in the result, I think it's consistent with line 96 right above?